http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -632,6 +632,84 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorReport"
+  "/api/v2/runtimes/{workspaceId}/{name}/start":
+    post:
+      summary: Start runtime (WIP).
+      description: >
+        Returns information about an existing azure runtime managed by Leo.
+        Poll this to find out when your runtime has finished starting up.
```

Comment:
What does the polling look like?

Reply:
oh, that's probably copy-paste error...will clean up